src/dune/docker.py

```diff
@@ -26,6 +27,16 @@ def __init__(self, container, image, cl_args):
                 self.execute_docker_cmd(
                     ['container', 'start', self._container])
             else:
+                # download dune image
```

Comment:
This seems to disable acquiring version when docker is NOT available/running.

Reply:
> This seems to disable acquiring version when docker is NOT available/running.

Fixed

> Additionally please add `--version-all` to report dune version, leap version, and cdt version.

Added following implementation:
mikel@msi:~/repo/DUNE$ ./dune --version-all
DUNE v1.0.0

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Listing...
leap/now 3.1.0 amd64 [installed,local]

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Listing...
cdt/now 3.0.0-1 amd64 [installed,local]